CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- next-header -->
+
+## [Unreleased] - ReleaseDate
+
+### Added
+
+- feat(build): default to `stable` build (#987)
```

Comment:
Can we teach cargo-release to add the URLs here?

```suggestion
- feat(build): default to `stable` build ([#987](https://github.com/ariel-os/ariel-os/pull/987))
```

Reply:
Yeah, I pushed something.